src/main/common/security/serviceTokenFactory.ts

```diff
@@ -0,0 +1,15 @@
+import IdamClient from 'idam/idamClient'
+
+import ServiceAuthToken from 'idam/serviceAuthToken'
+
+let token: ServiceAuthToken
```

Comment:
is this just cached for the request? or longer?

Reply:
Per app since when `serviceTokenFactory.ts` is imported Node.js caches imports and JS file is executed just once.